crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -826,56 +826,6 @@ mod test {
         handle.shutdown().await.unwrap().unwrap();
     }
 
-    #[tokio::test(flavor = "multi_thread")]
-    async fn gossip_info_request() {
```

Comment:
I can't see what the intention of this test was. I think it was testing the NetworkInfo but we're now able to test that elsewhere. We should test that calling it doesn't error but the values aren't really important at this level.